spec/views/application/_navbar.html.erb_spec.rb

```diff
@@ -3,6 +3,6 @@
 RSpec.describe "application/_navbar", type: :view do
   it "shows a login button" do
     render
-    expect(rendered).to have_link(t('devise.registrations.sign_in'), href: new_user_session_path)
```

Comment:
Why do you remove this?

Reply:
needs to be removed because ther is now an button with an image and no text-link anymore